src/main/scala/org/scalajs/tools/tsimporter/Trees.scala

```diff
@@ -146,8 +156,8 @@ object Trees {
   case class IndexMember(indexName: Ident, indexType: TypeTree, valueType: TypeTree) extends MemberTree
 
   case class PropertyMember(name: PropertyName, optional: Boolean,
-      tpe: TypeTree, static: Boolean) extends MemberTree
+      tpe: TypeTree, modifiers: Set[Modifier]) extends MemberTree
```

Comment:
I feel like we could use a `type Modifiers = Set[Modifier]` next to the definition of `Modifier`, to DRY up all these `Set[Modifier]`s.